Formula/quicktree.rb

```diff
@@ -3,7 +3,7 @@ class Quicktree < Formula
   homepage "https://www.sanger.ac.uk/science/tools/quicktree"
   url "https://github.com/khowe/quicktree/archive/v2.3.tar.gz"
   sha256 "3739f7962ce72c1d3c86ba0a3faa82cc60749f6d9f627c1aba3729b6c881dee4"
-  head "https://github.com/khowe/quicktree.git"
+  revision 1
```

Comment:
Please leave `head`. I prefer having it, and it only fails `brew audit` for new formula.